src/server/controllers/changePassword.ts

```diff
@@ -0,0 +1,131 @@
+import { Request } from 'express';
```

Comment:
general q - are there unit tests for the controllers in the express app or do we rely on the cypress suite for these?

Reply:
We rely on Cypress to integration test these, as these controllers/routes are only accessed through a web browser.